### PR TITLE
Fill page for gitcoin hackathon

### PR DESCRIPTION
**Pull request by DOBEN**
## Purpose

This page is hidden from the general documentation table of content but can be accessed as follows:

It summarized the information that needs to be conveyed to hackathon participants.

http://localhost:8000/net/guides/gitcoin-hackathon.html

### CODE REVIEW

**Review by dg-concordium — APPROVED** [review]
Maybe include a link to the general documentation site since this will be a hidden link and not visible in the TOC.

File: source/mainnet/net/guides/gitcoin-hackathon.rst [hunk 1]
```diff
@@ -4,7 +4,58 @@
...
 
 ============================
-Gitcoin hackathon essentials
+Gitcoin Hackathon Essentials
 ============================
 
-Here is information to help you get started with Concordium for the Gitcoin hackathon February 6 - 26, 2023.
+This page helps you to get started with Concordium for the Gitcoin hackathon February 6 - 26, 2023.
+
+With a total of $15K in prizes, it is the largest Concordium hackathon.
```
Comment (Bargsteen) [inline source/mainnet/net/guides/gitcoin-hackathon.rst:12]:
```suggestion
With a total of $15K in prizes, it is the largest Concordium hackathon ever held.
```

File: source/mainnet/net/guides/gitcoin-hackathon.rst [hunk 2]
```diff
@@ -4,7 +4,58 @@
...
+- 1st February @ 12:00 CET - Welcome, Concordium Overview & Intro to Rust
+- 2nd February @ 12:00 CET - Building Smart Contracts
+- 3rd February @ 12:00 CET - Building dApps & Ideation Session
+
+- 8th, 15th and 22nd February @ 9:00 - 17:00 CET - Weekly Developer Relations Office Hours
+
+You can watch the three workshops `live <https://www.airmeet.com/e/48fd9170-9344-11ed-87a2-15e85d06b129>`_.
+The materials from the workshop sessions are available on `GitHub <https://github.com/Concordium/voting-workshop>`_.
+
+Recording from previous workshops can be watched here:
```
Comment (Bargsteen) [inline source/mainnet/net/guides/gitcoin-hackathon.rst:29]:
```suggestion
Recordings from previous workshops can be watched here:
```

File: source/mainnet/net/guides/gitcoin-hackathon.rst [hunk 3]
```diff
@@ -4,7 +4,58 @@
...
+
+- `Building A Front-end Integration <https://www.youtube.com/watch?v=ux00M5XLHZk>`_.
+
+Quick guide
+===========
+
+- Install Rust
+- Install ``cargo concordium``
+- Install ``concordium-client``
+- Import a wallet key into ``concordium-client``
```
Comment (Bargsteen) [inline source/mainnet/net/guides/gitcoin-hackathon.rst:41]: Install browser wallet?

File: source/mainnet/net/guides/gitcoin-hackathon.rst [hunk 4]
```diff
@@ -4,7 +4,58 @@
...
+
+These setup steps are explained in detail :ref:`here<setup-env>`.
+An overview of the `developer documentation <https://developer.concordium.software/en/mainnet/index.html>`_ will help you to find the topics that you are interested in.
+
+.. note::
+
+    There is a public Concordium testnet node available at node.testnet.concordium.com on port 10000 (GRPCv1) and port 20000 (gRPCv2 and gRPC-web), if you don't want to sync your own node.
```
Comment (Bargsteen) [inline source/mainnet/net/guides/gitcoin-hackathon.rst:48]:
```suggestion
    There is a public Concordium testnet node available at ``node.testnet.concordium.com`` on port ``10000`` (gRPCv1) and port ``20000`` (gRPCv2 and gRPC-web). (You can also run your own node, but that will take some time to catch up to the newest blocks and thereby be usable).
```

**Review by Bargsteen — COMMENTED** [review]
Nice work 😊

File: source/mainnet/net/guides/gitcoin-hackathon.rst (hunk 4 above)
Comment (abizjak) [reply source/mainnet/net/guides/gitcoin-hackathon.rst:48]: I suggest linking to "how to run a node" on the text "run your own node".

Comment (abizjak) [inline source/mainnet/net/guides/gitcoin-hackathon.rst:48]:
```suggestion
    There is a public Concordium testnet node available at ``node.testnet.concordium.com`` on port 10000 (GRPCv1) and port 20000 (gRPCv2 and gRPC-web), if you don't want to sync your own node.
```

**Review by abizjak — APPROVED** [review]

File: source/mainnet/net/guides/gitcoin-hackathon.rst (hunk 4 above)
Comment (DOBEN) [reply source/mainnet/net/guides/gitcoin-hackathon.rst]: Added a link to the `run your own node` section.

**Pull request merged**